.github/workflows/ci.yml

```diff
@@ -1,7 +1,7 @@
 name: CI
 on: [push, pull_request]
 env:
-  dockerid: ${{ secrets.dockerid }}
+  dockerid: ${{ secrets.dockerid || 'openrct2' }}
```

Comment:
I'll check if this dockerid is indeed the repo name and add a comment. If it is - there's no need for this to be secret

Reply:
No, this is not correct. I was considering leaving it secret, but it can be read from hub.docker.com. This is `intelorca` currently and it should stay.